apps/docs/components/MDX/auth_error_codes_table.mdx

```diff
@@ -55,6 +55,8 @@ To supplement HTTP status codes, Supabase Auth returns a string error code which
 | `provider_email_needs_verification`     | Not all OAuth providers verify their user's email address. Supabase Auth requires emails to be verified, so this error is sent out when a verification email is sent after completing the OAuth flow.                                                                                                         |
 | `reauthentication_needed`               | A user needs to reauthenticate to change their password. Ask the user to reauthenticate by calling the `supabase.auth.reauthenticate()` API.                                                                                                                                                                  |
 | `reauthentication_not_valid`            | Verifying a reauthentication failed, the code is incorrect. Ask the user to enter a new code.                                                                                                                                                                                                                 |
+| `refresh_token_not_found`               | Session containing the refresh token not found.                                                                                                                                                                                                                                                               |
```

Comment:
Typically happens when the user has logged out. Not sure what the action here should be though